samples/graph-activity-feed-broadcast/csharp/README.md

```diff
@@ -91,7 +91,7 @@ This sample shows a feature where user can broadcast a message to all members of
     Set a redirect URI:
     * Select **Add a platform**.
     * Select **web**.
-    * Enter the **redirect URI** for the app in the following format: `https://{Base_Url}/Auth/end`, `https://{Base_Url}/Auth/Start`. This will be the page where a successful implicit grant flow will redirect the user.
+    * Enter the **redirect URI** for the app in the following format: `https://{Base_Url}/Auth/End`, `https://{Base_Url}/Auth/Start`. This will be the page where a successful implicit grant flow will redirect the user.
     Enable implicit grant by checking the following boxes:  
     ✔ ID Token  
```

Comment:
Can you please add the doc link for sending bulk notification below in further reading section ?

Reply:
Added